plugin-server/src/worker/ingestion/event-pipeline/3-pluginsProcessEventStep.ts

```diff
@@ -13,7 +15,7 @@ export async function pluginsProcessEventStep(
     let processedEvent: PluginEvent | null = event
 
     // run processEvent on all events that are not $snapshot
-    if (event.event !== '$snapshot') {
+    if (!EVENTS_TO_IGNORE.includes(event.event)) {
```

Comment:
Do we need to do anything in the previous pipeline step? Presumably this is also relevant there.

I would love to see these performance events in a separate `topic` with a separate consumer + consumer group so we have some isolation of the two workloads. Happy to pair on this.

Reply:
I don't know, do we? 😅  

Do you see the separate topic as a blocker? Feels like a good call but possibly quite high effort?

Reply:
It's more than no effort but not much.

I don't know the shape of the performance events so much in terms of schema, volume or functionality so not sure how these events will effect e.g. batching strategy for processing. Batching depends on distinct_id. Not sure if it's relevant to these events or not.

It's probably fine (tm) and we'll find out soon enough. fwiw pipeline has a target of 30 seconds from ingestion to insightable. Happy to merge this in though and we can review after, and put in some guidelines for next time we introduce a new data stream.